style.css

```diff
@@ -0,0 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap');
+
```

Comment:
# Mobile Size: Abou
- Please try to keep each section inside the block and not overtake another block. 
- **See your Job**:
- 
![Capture](https://user-images.githubusercontent.com/63972670/203863185-70c591b7-fbbd-4b60-ac13-05b849b569e9.PNG)
-
- 
**Have a look at the Original design:**
![Capture1](https://user-images.githubusercontent.com/63972670/203863362-d4b7d066-f749-441d-ab3f-7e2ae890683c.PNG)

Reply:
Thanks for the suggestions I have fixed the issue